pinot-segment-local/src/main/java/org/apache/pinot/segment/local/io/writer/impl/FixedByteChunkSVForwardIndexWriter.java

```diff
@@ -101,6 +103,12 @@ public void putDouble(double value) {
     flushChunkIfNeeded();
   }
 
+  public void putBigDecimal(BigDecimal value) {
```

Comment:
We should not hit this method because big-decimal shouldn't be stored as fixed length